src/Controls/src/Core/Visuals/VisualTypeConverter.cs

```diff
@@ -27,16 +27,8 @@ public override bool CanConvertTo(ITypeDescriptorContext context, Type destinati
 		void InitMappings()
 		{
 			var mappings = new Dictionary<string, IVisual>(StringComparer.OrdinalIgnoreCase);
-			Assembly[] assemblies = AppDomain.CurrentDomain.GetAssemblies();
-
-			// Check for IVisual Types
-			foreach (var assembly in assemblies)
-				Register(assembly, mappings);
-
-			if (Internals.Registrar.ExtraAssemblies != null)
-				foreach (var assembly in Internals.Registrar.ExtraAssemblies)
-					Register(assembly, mappings);
 
+			Assembly[] assemblies = AppDomain.CurrentDomain.GetAssemblies();
 
 			// Check for visual assembly attributes	after scanning for IVisual Types
 			// this will let users replace the default visual names if they want to
```

Comment:
This comment could use some updating, since we don't "scan for IVisual Types" anymore, so it isn't checked "after" anymore.